crypto/rsa/rsa_sign.c

```diff
@@ -7,6 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
+/*
+ * Some low level APIs are deprecated for public use, but still ok for
+ * internal use.
+ */
+#include "internal/deprecated.h"
```

Comment:
AFAICT this just seems to be for the various DIGEST_LENGTH macros. Perhaps we don't deprecate them?

Reply:
I'm not sure which way we should go.  Deprecating them seems like the right thing to do to reduce the APIs but they are never going to change.

The lengths can be determined via EVP, not that it helps here.

Reply:
Maintaining them seems ok to me and even quite useful. There's no reason to break code that just uses these defines.

Reply:
Okay, I'll update all of these in the morning.

Reply:
Updated here, will do the other PRs soon.